### DISCUSSION TITLE
Looking for collaborators!

### DISCUSSION THREAD

**Discussion by Ismoh** [Ismoh/local-lua-debugger-vscode#6] [Announcements]
Unfortunatelly local-lua-debugger-vscode initial designed and implemented by @tomblind is not maintained anymore.

I'll publish a version 2.0 on vscode marketplace soon, when there are minimum four volunteer collaborators, as I don't want this fork to be unmaintained aswell.

If you are interested in beeing a collaborator, just comment this announcement!

Happy coding!

**Comment by sewbacca** [comment]
I am using the extension on a daily basis and have written the closure lookup PR. While I am not well versed in TypeScript itself, I think I could look over incoming PRs and maybe add a feature or two, if I get around to it.

**Reply by Ismoh** [reply]
Awesome! I added you as a collaborator! Looking forward for PRs. I'll take today to set up vscode marketplace.

**Comment by ogsadmin** [comment]
Hi @Ismoh - I am also mainly a user of the extension, but I believe, that it is good thing to keep it alive. So yes, I will try to help!

**Reply by Ismoh** [reply]
@ogsadmin your collaborator invite was expired. I sent a new one right now. Check your inbox :)

**Comment by eduardomezencio** [comment]
I'm a user of the extension, I can help but I'm not really experienced in javascript/typescript or vscode extensions. I actually use this extension in neovim only now.

I'm trying to improve a little the experience configuring this extension, because having to put the path to the extension folder in the configuration is not nice. I'm sending a pull request so you guys can take a look

**Reply by sewbacca** [reply]
That would be nice; I recently started using neovim as well, so I will gladly be taking a look at this. I also have managed setting up the debugger, but some things are still quirky.

Anyways I'll be looking forward for your PR.

**Reply by eduardomezencio** [reply]
https://github.com/Ismoh/local-lua-debugger-vscode/pull/16 and https://github.com/Ismoh/local-lua-debugger-vscode/pull/17 (edited because I have separated my previous pull request in two and closed the old one)

**Reply by Ismoh** [reply]
Thanks for the PR!
I am currently out of order, as there were private changes.
I'll can add you as collaborator, @eduardomezencio , if you like?
